test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,15 +1,43 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
 import java.util.Arrays;
-import java.util.List;
 
 import org.junit.Test;
 
+import static org.junit.Assert.*;
```

Comment:
Avoid using '*' imports. It is always recommended to import only what is needed.